serve.sh

```diff
@@ -1,9 +0,0 @@
-PORT=$1
```

Comment:
i.e. to debug tests in one browser (Chrome): `karma start karma.conf.js --browsers=Chrome --single-run=false`